processing/src/main/java/io/druid/query/groupby/epinephelinae/RowBasedGrouperHelper.java

```diff
@@ -318,6 +318,22 @@ public RowBasedKeySerdeFactory(DateTime fudgeTimestamp, boolean sortByDimsFirst,
     // Entry in dictionary, node pointer in reverseDictionary, hash + k/v/next pointer in reverseDictionary nodes
     private static final int ROUGH_OVERHEAD_PER_DICTIONARY_ENTRY = Longs.BYTES * 5 + Ints.BYTES;
 
+    private static final Comparator<RowBasedKey> ROW_BASED_KEY_DIM_COMPARATOR = new Comparator<RowBasedKey>()
```

Comment:
Why not just a static method for comparing RowBasedKeys in `RowBasedKeySerde` and then calling this method from comparators, returned from `entryComparator()`? Wrapping this functionality into another comparator object seems needless to me.